cluster/ci/config/prow/jobs/cluster-launch-e2e.yaml

```diff
@@ -183,6 +186,18 @@ objects:
           -e openshift_test_repo=${RPM_REPO} -e 'oreg_url=${IMAGE_FORMAT}' \
           playbooks/${CLUSTER_TYPE}/openshift-cluster/launch.yml
 
+        # if we need to publish additional images like jenkins images, as those
+        # images are also part of what we want to verify with this e2e run, then
+        # update the cluster accordingly
+        if [[ "${POST_CLUSTER_IMAGE_UPDATE}" == "true" ]]; then
+           ansible-playbook -vv \
+           # POST_CLUSTER_IMAGE_UPDATE_IMAGE_FORMAT
+           # should be set to something like registry.svc.ci.openshift.org/<test namespace>/<promoted image stream for each jenkins image>
+           -e 'docker_image_url=${POST_CLUSTER_IMAGE_UPDATE_IMAGE_FORMAT}' \
+           -e 'openshift_ist=${POST_CLUSTER_IMAGE_UPDATE_DEST_IST}' \
+           playbooks/openshift-imagestreamtags/update_imagestreamtags.yml
+        fi
```

Comment:
i'll just reiterate for sake of the discussion that i'd really really prefer that the cluster endpoint+credentials were just made available here so people could just invoke oc commands directly rather than having to introduce new ansible roles to accomplish this.

Reply:
alternatively maybe we introduce a generic ansible role that is "run an oc command" where the command is provided as an ansible variable.  At least that means we don't have to keep introducing new roles.

Reply:
@smarterclayton if we consider the actions here as the first part of a test suite that orients itself with a `$KUBECONFIG` there is no need to change this template and the test script in their repo can set this up -- what's wrong with an approach like that?